README.md

```diff
@@ -195,6 +195,20 @@ You can change Graphile address and port using the flags graphile-url.
 /nonodo --graphile-url http://mygraphileaddress:5034
```

Comment:
I think it's better to remove the `/`.

Reply:
<img width="464" alt="image" src="https://github.com/user-attachments/assets/688f0391-a975-403a-8e13-96d587676a89">

It's nice to show the endpoint in the log.

./nonodo --salsa

Http Rollups for development started at http://localhost:5004
**Lambada running at http://localhost:5005**
GraphQL running at http://localhost:8080/graphql
Inspect running at http://localhost:8080/inspect/
Press Ctrl+C to stop the node

[16:42:33.621] INF nonodo: ready after=1.895389s